libs/backend/src/cast_vote_records/export.ts

```diff
@@ -133,6 +133,21 @@ export interface ResultSheet {
   readonly batchLabel?: string;
```

Comment:
sidebar: is this still needed?

Reply:
I'm not entirely sure - left a note to revisit this in my temporary catch-all issue: https://github.com/votingworks/vxsuite/issues/3996